src/cart/crt_rpc.c

```diff
@@ -208,6 +208,25 @@ static struct crt_proto_rpc_format crt_fi_rpcs[] = {
 
 #undef X
 
+#define X(a, b, c, d, e) case a: return #a;
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  Macros with flow control statements should be avoided
(style)  macros should not use a trailing semicolon